build-tools/packages/build-cli/src/flags.ts

```diff
@@ -36,14 +40,36 @@ export const packageSelectorFlag = Flags.build({
     description: "package",
     // Can't be used with release groups.
     exclusive: ["g"],
+    multiple: false,
+    parse: async (input) => {
+        const split = input.split("=");
+        const dep = split[0];
+        const version = split[1];
+        return { dep, version };
+    },
 });
 
 /**
  * A re-usable CLI flag to parse bump types.
  */
 export const bumpTypeFlag = Flags.build({
     char: "t",
-    description: "bump type",
+    description: "Version bump type.",
     options: ["major", "minor", "patch", "current"],
-    required: true,
+    parse: async (input) => {
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+        if (isVersionBumpTypeExtended(input)) {
+            return input;
+        }
```

Comment:
Explicit return for the other case?

Reply:
Interesting. I think that rule will conflict with our explicit return types rule in `recommended`. Does it still complain if you set an explicit return type on the function?